vector/src/withpinning/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <!-- Do not allow clearText traffic -->
+    <base-config cleartextTrafficPermitted="false">
+        <trust-anchors>
+            <certificates src="system" />
```

Comment:
@bmarty I will check whether the app with pinning should trust or not the pre-installed system CAs before merging your PR.
I agree it was the case until now (because it is the default behavior), but I need to check whether this is the expected behavior...